.github/ISSUE_TEMPLATE/proposal.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: Proposal
 about: A template for major Druid change proposals
-title: "[PROPOSAL]"
```

Comment:
This part looks fine, I think we agreed in the mailing list that the tags are not needed.